Gemfile

```
@@ -4,5 +4,5 @@ gem 'foreman'
 gem 'govuk_app_config'
 gem 'rack'
 gem 'redis-namespace'
-gem 'govuk_sidekiq', require: false
+gem 'govuk_sidekiq', "~> 5.0.0", require: false # Temp: pinned to avoid dependabot automatically upgrading as we'll need to do some work here to upgrade
```

```suggestion
gem 'govuk_sidekiq', "< 6", require: false # Temp: pinned to avoid dependabot automatically upgrading as we'll need to do some work here to upgrade
```

Nit: this might better express our intentions. `~> 5.0.0` suggests we want `< 5.1`